specs/span-batches.md

```diff
@@ -201,10 +206,12 @@ Our goal is to find the sweet spot on code complexity - span batch size tradeoff
 I decided that using RLP for all variable length fields will be the best option,
 not risking codebase with gnarly custom encoding/decoding implementations.
 
-### Store `y_parity` instead of `v`
+### Store `y_parity` and `protected_bit` instead of `v`
 
-For legacy type transactions, `v = 2 * ChainID + y_parity`. For other types of transactions, `v = y_parity`.
-We may only store `y_parity`, which is single bit per L2 transaction.
+Only legacy type transactions can be optionally protected. If protected([EIP-155]), `v = 2 * ChainID + y_parity`.
```

Comment:
```suggestion
Only legacy type transactions can be optionally protected. If protected([EIP-155]), `v = 35 + 2 * ChainID + y_parity`.
```

It's offset by 35: https://github.com/ethereum/EIPs/blob/master/EIPS/eip-155.md#specification

Reply:
Thanks. Fixed!